adminpages/reports/sales.php

```diff
@@ -404,17 +404,44 @@ function drawVisualization() {
 
 			var data = google.visualization.arrayToDataTable([
 				[
-					'<?php echo esc_html( $date_function );?>', '<?php _e( 'Renewals', 'paid-memberships-pro' );?>', '<?php echo esc_html( sprintf( __( 'New %s', 'paid-memberships-pro' ), ucwords( $type ) ) );?>', '<?php _e( 'Average*', 'paid-memberships-pro' );?>', 
+					{ 'label': '<?php echo esc_html( $date_function );?>', 'type': 'string' },
```

Comment:
The correct way to output dynamic data to JS and prevent single quote / JS breakage is to create the data in PHP and then echo the value as JSON.

The code here looks like it would present a JS break if someone translates text with single quotes in them.

Reply:
Something like `<?php echo wp_json_encode( $data ); ?>`